BubblesKotlin/app/src/main/AndroidManifest.xml

```diff
@@ -51,24 +52,36 @@
                     android:pathPattern="/chat/*"
                     android:scheme="https" />
             </intent-filter>
+            <!--
+            -->
```

Comment:
Missing comment?

Reply:
Done. I meant to add a comment and forgot about it.